CHANGES.rst

```diff
@@ -172,6 +172,12 @@ Bug Fixes
 
 - ``astropy.utils``
 
+  - Fix to allow the C-based _fast_iterparse() VOTable XML parser to
+    relloc() its buffers instead of overflowing them.  [#5824] [#5869]
```

Comment:
I would suggest to remove the parenthesis, we don't usually detail the bug they fix, for more info people can go to the issues referenced.